Makefile

```
@@ -3,15 +3,15 @@ export GO111MODULE=on
 .PHONY: all build test clean distclean
 
 build:
-	go build
+	go build -o spiffe-helper cmd/spiffe-helper/*.go
```

Using go build in this way (building specific .go files) is not recommended. You should instead build the package:

```suggestion
	go build -o spiffe-helper ./cmd/spiffe-helper
```